deploy/internal/deployment-endpoint.yaml

```diff
@@ -4,6 +4,8 @@ metadata:
   labels:
     app: noobaa
   name: noobaa-endpoint
+  annotations:
+    ConfigMapMD5: ""
```

Comment:
Why we call it `ConfigMapMD5` if it uses sha256?

Reply:
Changed that to `ConfigMapHash`